firestore/angular-rewrite/README.md

```diff
@@ -18,4 +18,8 @@ Run `ng build` to build the project. The build artifacts will be stored in the `
 
 ## Further help
 
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
\ No newline at end of file
+To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
+
+## Running the Emulators
+The command to start the local emulators is:
+`firebase emulators:start --import./exported-firestore --export-on-exit=./exported-firestore`
```

Comment:
nit: Let's place --export-on-exit in square brackets - this one is optional and leaving it off means the developer will get a fresh dataset each execution